trin-history/src/lib.rs

```diff
@@ -113,12 +113,11 @@ pub fn spawn_history_network(
 pub fn spawn_history_heartbeat(network: Arc<HistoryNetwork>) {
     tokio::spawn(async move {
         let mut heart_interval = interval(Duration::from_millis(30000));
+        heart_interval.tick().await;
 
         loop {
-            let radius = network.overlay.store.read().radius();
-            // calculate the percentage of the whole data ring covered by the data radius
-            let coverage_percent = radius.byte(31) as f32 * 100.0 / 255.0;
-            info!("report: radius={coverage_percent:.1}%");
+            let storage_log = network.overlay.store.read().get_summary_info();
+            info!("storage-report: {storage_log}");
             heart_interval.tick().await;
```

Comment:
*nit* if you want to get rid of an extra line of code here, it seems like you could just move `heart_interval.tick().await;` to the beginning of the `loop { .. }`

Reply:
Oh yeah, it evolved in a weird way, because I didn't want to wait 30s to get the first log line, but there is some weird thing (bug?) where the first tick expires roughly immediately (at least on my runs). 👍🏻 for simplifying, though I'll add a comment so I may not kill a line.